### ISSUE TITLE
Select + Variant (listbox) -> Not Defaulting Back to Selected Value on Form Validation

### ISSUE THREAD

**Issue by WorthLMS** [livewire/flux#60]
Replication:

Created a flux:select with a couple of sample options from using the listbox variant.
wire:model the flux:select

triggered validation error on another field (component refreshes)...

Selected Value of select is gone.

flux:input values remain, but flux:select with variant listbox resets to null

**Comment by WorthLMS** [comment]
Confirmed Findings.

Removed Searchable (Issue Remained)
Removed variant="listbox" (normal select), value was retained after validation error
Replaced variant="listbox" value was NOT retained after validation error

**Comment by gaiustemple** [comment]
Also having this issue on an edit page with a "listbox" variant select, which tries to load the previously saved value into the select - but just shows blank.

**Comment by patrykszady** [comment]
Similar with `variant="combobox"`

**Comment by jsandfordhughescoop** [comment]
Same thing happening here with anything but the default variant

**Comment by patrykszady** [comment]
Hi, so when I use the `combobox` select my pre-selected value does not populate like it does on a regular `select`. Can anyone replicate this?
 ```        <flux:select label="Vendor" wire:model.live="form.vendor_id" variant="combobox" placeholder="Choose vendor...">
            @foreach($vendors as $vendor)
                <flux:option>{{$vendor->name}}</flux:option>
            @endforeach
        </flux:select>```

**Comment by jsandfordhughescoop** [comment]
> Hi, so when I use the `combobox` select my pre-selected value does not populate like it does on a regular `select`. Can anyone replicate this?
> 
> ```
> <flux:select label="Vendor" wire:model.live="form.vendor_id" variant="combobox" placeholder="Choose vendor...">
>             @foreach($vendors as $vendor)
>                 <flux:option>{{$vendor->name}}</flux:option>
>             @endforeach
>         </flux:select>
> ```

Yes this is an issue at the moment

**Comment by patrykszady** [comment]
Caleb on issue #60:
“ Thank you for the report. This issue has been fixed and will be available in the next release.”

Please close this once confirmed fixed in 1.0.8

**Comment by calebporzio** [comment]
This should be fixed, however, if it is not, please re-submit with a copy/pastable example.
